README.md

```diff
@@ -1,2 +1,19 @@
 # rest
-A kitchen simulator
+
+![GitHub repo size](https://img.shields.io/github/repo-size/Lcv004/rest)
```

Comment:
What are these URLs?

Reply:
Those are known as badges or shields for Github. These links refers to "a service for concise, consistent, and legible badges in SVG and raster format, which can easily be included in GitHub readmes or any other web page". With these I can easily check some metrics of my project in the README.md in real time. Metrics as the size of the project, number of opened issues or PRs, and so on.

For instance, check the README.md of this repository:
https://github.com/badges/shields

Reply:
Cool thanks!